parsers/markdown.c

```diff
@@ -28,6 +28,7 @@
 #include "parse.h"
 #include "read.h"
 #include "vstring.h"
+#include "utf8_str.h"
```

Comment:
The changes for parsers/markdown.c should be a part of the second commit.

Reply:
fixed